### ISSUE TITLE
[Question]How to get rte selection inside the result function

### ISSUE THREAD

**Issue by SeongwoonHong** [artf/grapesjs#1702]
```
rte.add('custom-link', {
          icon: '<div><img src="assets/images/link.png" /><input name="link" id="custom-link-input" value="" /></div>',
          attributes: {title: 'Link', class: 'gjs-rte-action'},
          event: 'change',
          result: (rte) => {
            console.log(`<a href="#">${rte.selection()}</a>`)
          },
          update: (rte, action) => {
            // ...
          }
        });
```
I'm basically trying to put an input fieldinside the RTE so that users can make link more easily.
console.log(`<a href="#">${rte.selection()}</a>`) inside the result function is returning an empty anchor tag like this <a href="#></a>.
I've tried everything.. even store the rte.selection() value to the global variable inside the update function (as it seems like update function has the correct value) and reuse inside the result function but none of them worked. 
so basically,
1) click any text and you see the icon and input field.
2) type any text inside the input field
3) blur the focus so that result function is triggered.
4) see the console and it says <a href="#></a>
for now, href is always # cuz i will add value later on. but the problem is rte.selection() is empty..
your help would be really appreciated. thank you
